### DISCUSSION TITLE
eBird

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-Pi#481] [Announcements]
Please don't send your data to eBird

My best regards,
Patrick

**Comment by DD4WH** [comment]
Thanks Patrick !
I would like to clear this up a little bit more:
* In order for bird distribution data to be useful for scientific purposes, every single bird record has to be validated in the same way. At the moment, every bird record in eBird has been validated by a human (you have seen the bird with your eyes or you have heard the bird with your ears)
* Every single bird detection produced with artificial intelligence methods/machine learning (like BirdNET-Pi) has to be validated by a human by listening to the recording and by comparing it with existing reference recordings. But even that is not enough! You have to ensure that you also checked the plausibility that the bird species detected can be expected in the kind of habitat you recorded in and the probability that this bird species occurs in the specific biogeographic region you are in. 
* Only after you performed these validations, it makes sense to put that record into a bird distribution database like eBird

So, please do NOT send UNVALIDATED bird data to eBird or any other bird portal.

ALWAYS validate BirdNET detections by listening and comparing your recordings to reference recordings and check the plausibility using known habitat preferences and known biogeographic distribution data of the species, before you contribute to a bird data portal. [of course, this also applies to any other automatic bird song detection method, like Merlin or Kaleidoscope or etc . . .]

Only by doing this, we can contribute valuable data to the existing scientific data collections on bird distribution. Thank you all in advance !

Best regards,
Frank Dziock DD4WH

**Comment by DD4WH** [comment]
In the "Feature requests" thread several users requested a feature to automatically generate a list of bird detections to submit that list to bird data portals like eBird. I would like to comment on that request and try to collate here arguments against such a feature:

* ALL bird data portals have a common goal: they collect data from professional and volunteer ornithologists 
* the purpose of this data collection is: gather distribution data and habitat preference data and phenological data on bird species and use this data for SCIENTIFIC purposes, so to expand our general knowledge and understanding of bird biology and ecology

The data collection for this purpose has to be strictly standardized:

* the method for collecting bird sightings must be comparable --> in most cases people go out in the field and listen for bird songs and look around for bird sightings
* the time in the field and the amount of concentration one person can spend in the field is limited. Also, people count individual birds and put those bird species numbers into the bird portals´ protocol

Now, consider an automatic system like BirdNET-Pi:
* it is active and concentrated 24hours, 7 days a week
* it only takes into account bird sound, it does never take into account sightings
* BirdNET-Pi does NOT count individuals, it does count detections (every three seconds)
* BirdNET-Pi has more false IDs than a human observer, especially in an urban environment with many noise sounds that have not been involved in BirdNETs´ training process

By this comparison of a human observer and an automatic system, it becomes pretty clear, that collating BirdNET-Pi data in an automatic way and sending these data to a portal like eBird does not make sense. This is because the method the data was collected is by no means comparable. And this even is true, if ALL detections have been double-checked by manual ID by ear.

Example: 
Common Buzzard calls for one hour while flying over my house.
* BirdNET-Pi has more than 300 detections of Common Buzzard and puts these into a table (please note that ALL of the 300 detections would have to be verified by the user by ear!)
* a human observer hears and sees one buzzard individual and reports one buzzard to the portal

The data portal compares the two sites and the conclusion is, that one site has 300 times more buzzards than the other.
And this clearly does not make sense.

So, in conclusion, please do not report BirdNET-Pi detections automatically to any bird data portal, EVEN if you identified every single detection by ear.

This helps to keep the data on these portals comparable and clean and scientifically useful.

I guess that -in the near future- bird data portals will have a special protocol where you can upload such automatically collated and manually verified data. But this data has to be kept separate from all the other protocols for the reasons stated above.

Thanks for sticking to this rule, which will help save the data quality in bird portals !

Best regards,

Frank Dziock DD4WH
